app/main.py

```diff
@@ -1,8 +1,19 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people: dict) -> None:
+    person_list = [Person(person["name"], person["age"]) for person in people]
+
+    for person in people:
+        person_instance = Person.people[person["name"]]
+        if "wife" in person and person["wife"]:
+            person_instance.wife = Person.people[person["wife"]]
+        elif "husband" in person and person["husband"]:
```

Comment:
The same comment as above
```suggestion
        elif person.get("husband"):
```